plugins/outputs/cloudwatchlogs/cloudwatchlogs.go

```diff
@@ -38,8 +39,11 @@ const (
 	metricRetryTimeout = 2 * time.Minute
 
 	attributesInFields = "attributesInFields"
+	containerInsightRegexp = "^/aws/.*containerinsights/.*/(performance|prometheus)$"
 )
 
+var ciCompiledRegexp, _ = regexp.Compile(containerInsightRegexp)
```

Comment:
I see there is a `prometheus_config_linux.json` file with this:
```
{
  "agent": {
    "region": "us-east-1"
  },
  "logs": {
    "metrics_collected": {
      "prometheus": {
        "cluster_name": "TestCluster",
        "log_group_name": "/aws/ecs/containerinsights/TestCluster/prometheus",
        "prometheus_config_path": "file:/tmp/prometheus.yaml",
        "ecs_service_discovery": {
          "docker_label": {
            "sd_job_name_label": "ECS_PROMETHEUS_JOB_NAME_1",
            "sd_metrics_path_label": "ECS_PROMETHEUS_METRICS_PATH",
            "sd_port_label": "ECS_PROMETHEUS_EXPORTER_PORT_SUBSET"
          },
```
But I also see docs say the `log_group_name` is optional:
>log_group_name— specifies the log group name for the scraped Prometheus metrics. This field is optional. If you omit it, CloudWatch uses /aws/containerinsights/cluster_name/prometheus for logs from Amazon EKS and Kubernetes clusters.